azure_runner.yml

```diff
@@ -1,8 +1,10 @@
 name: AzureRunner
 channels:
   - defaults
+  - pytorch
 dependencies:
   - pip=20.1.1
+  - pytorch=1.6.0
```

Comment:
this is not good - this means job submission in the builds will take lots longer. Can you re-work the  dependencies, or use local torch imports to get that back out?